content/docs/command-reference/import-url.md

```diff
@@ -125,10 +124,9 @@ source.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create the import `.dvc` file without actually downloading
-  `url`. The data hash is not calculated when this option is used, only the
-  import metadata is saved to the `.dvc` file. You can use `dvc commit` to
-  finish the operation.
+- `--no-exec` - create the import `.dvc` file but don't download `url` (assumes
+  that the data source is valid). You can use `dvc commit` to finish the
```

Comment:
If we remove mention of existing outputs, `dvc commit` looks artificial. It's not clear why would you run it.

I still think we can fit into one paragraph, mention existing files (avoid downloads, avoid run in case of `dvc run`

Reply:
Wait yeah good point, `commit` doesn't finish the operation (that would be `update` to download it). So I would argue to remove the reference to `commit` here. Again, is import + commit an actual case we want to document or just a side-effect?

Reply:
Separately, I wonder why repro doesn't have --no-exec and why import* don't have --no-commit. Is it worth asking on the core repo?

Reply:
> I still think we can fit into one paragraph

Done, PTAL.